lib/ts/recipe/dashboard/api/user/create/passwordlessUser.ts

```diff
@@ -57,39 +65,71 @@ export const createPasswordlessUser = async (
 
     if (
         email !== undefined &&
-        (passwordlessRecipe.config.contactMethod === "EMAIL" ||
-            passwordlessRecipe.config.contactMethod === "EMAIL_OR_PHONE")
+        (passwordlessOrThirdpartyPasswordlessRecipe.config.contactMethod === "EMAIL" ||
+            passwordlessOrThirdpartyPasswordlessRecipe.config.contactMethod === "EMAIL_OR_PHONE")
     ) {
         email = email.trim();
-        const validateError = await defaultValidateEmail(email);
-        if (validateError !== undefined) {
+        let validationError: string | undefined = undefined;
+
+        if (passwordlessOrThirdpartyPasswordlessRecipe.config.validateEmailAddress !== undefined) {
+            validationError = await passwordlessOrThirdpartyPasswordlessRecipe.config.validateEmailAddress(
+                email,
+                tenantId
+            );
+        } else {
+            validationError = await defaultValidateEmail(email);
```

Comment:
this is wrong.. if validateEmailAddress us undefined, then the contact method was phone number, so it should never come here. So you can use `passwordlessOrThirdpartyPasswordlessRecipe.config.validateEmailAddress!(...)`

Reply:
done